lib/geordi/commands/shell.rb

```diff
@@ -2,12 +2,16 @@
 long_desc <<-LONGDESC
 Example: `geordi shell production`
 
-Lets you select the server to connect to when called with `--select-server`:
+Lets you select the server to connect to from a menu when called with `--select-server` or the alias `-s`:
 
     geordi shell production -s
+
+Lets you select the server to connect to directly by passing the number:
```

Comment:
In a way, this is "selecting" as well, but I'm not sure everyone will understand. What about: "If you already know the number of the server you want to connect to, just pass it along:"

Again, if `-s2` works, please use the syntax here.

Reply:
Please remember to update the README when changing this.

Reply:
Please also add this description to console.rb.